distribution/server-dist/src/main/docs/examples/map-storage-concurrenthashmap.cli

```diff
@@ -0,0 +1,14 @@
+embed-server
+
+/subsystem=keycloak-server/spi=authenticationSessions:add(default-provider=map)
+/subsystem=keycloak-server/spi=client:add(default-provider=map)
+/subsystem=keycloak-server/spi=clientScope:add(default-provider=map)
+/subsystem=keycloak-server/spi=group:add(default-provider=map)
+# /subsystem=keycloak-server/spi=realm:add(default-provider=map)
+/subsystem=keycloak-server/spi=role:add(default-provider=map)
+/subsystem=keycloak-server/spi=user:add(default-provider=map)
+
+/subsystem=keycloak-server/spi=mapStorage:add(default-provider=concurrenthashmap)
+/subsystem=keycloak-server/spi=mapStorage/provider=concurrenthashmap:add(properties={dir="${jboss.server.data.dir}/map"},enabled=true)
```

Comment:
We should disable the UserFederation provider.

Reply:
Looking through the code, it's not that easy. Disabling `jpa` provider leads to an exception on startup, so I'd rather keep it as is - map storage it is a preview feature and the user federation will be updated to tree store soon anyway.